apps/package.json

```diff
@@ -188,9 +188,17 @@
     "which": "1.1.1"
   },
   "dependencies": {
+    "bootstrap": "^3.3.7",
     "filesaver.js": "0.2.0",
+    "jquery-mask-plugin": "^1.14.8",
+    "jquery-placeholder": "^2.3.1",
+    "jquery-query-object": "^2.2.3",
+    "jquery-ujs": "^1.2.2",
     "jszip": "3.0.0",
     "karma-coveralls": "^1.1.2",
-    "query-string": "4.1.0"
+    "qtip2": "^3.0.3",
+    "query-string": "4.1.0",
+    "selectize": "^0.12.4",
+    "timeago": "^1.5.4"
```

Comment:
these library versions don't all match up with what we were using before, so this might not be a great idea.